packages/graphql_gateway/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = require('../../packages/config/eslint')({
```

Comment:
Is it not possible to add the config package in the package.json and reference it like a normal package instead of using a relative path?

Reply:
found a way
but we need either two configs or a deep merge as now, will move to two configs